core/templates/pages/learner-dashboard-page/learner-dashboard-icons.component.spec.ts

```diff
@@ -724,7 +724,7 @@ describe('Learner Dashboard Icons Component', () => {
       spyOn(
         learnerDashboardActivityBackendApiService,
         'removeFromLearnerPlaylistModal')
-        .and.returnValue(null);
+        .and.returnValue();
```

Comment:
Do we even need this, since we won't return anything?

Also if we really need this reformat this to
```suggestion
      ).and.returnValue();
```

Reply:
Using `callthrough()` now. Done